tasks/models.py

```diff
@@ -53,6 +53,29 @@ def filter_by_symbol(cls, priority_filter):
         print(priority_filter)
         return cls.objects.filter(priority=priority_filter)
 
+    @classmethod
+    def create_task(cls, title, assignee, assigner, priority, status, description):
+        if not title or title == "":
+            raise ValueError("Must add title")
+        if not isinstance(priority, Priority):
+            raise ValueError("Must use Priority enum")
+        if not isinstance(status, Status):
+            raise ValueError("Must use Status enum")
+        assigner_role = assigner.role
+        assigner_team = assigner.team_id
+        assigne_team = assignee.team_id
```

Comment:
```suggestion
```

Reply:
Already checked in `Task.objects.create`

Reply:
Didn't remove the blank title since Django does not check that.